src/RestApi/StoreApi/Schemas/CartItemSchema.php

```diff
@@ -322,14 +329,14 @@ public function get_item_response( $cart_item ) {
 	}
 
 	/**
-	 * If a product has low stock, return the remaining stock amount for display.
+	 * Returns the remaining stock for a product if it has stock.
 	 *
-	 * Note; unlike the products API, this also factors in draft orders so the results are more up to date.
+	 * This also factors in draft orders.
 	 *
 	 * @param \WC_Product $product Product instance.
 	 * @return integer|null
 	 */
-	protected function get_low_stock_remaining( \WC_Product $product ) {
+	protected function get_remaining_stock( \WC_Product $product ) {
```

Comment:
This isn't needed now right?

Reply:
It abstracted out the logic from `get_low_stock_remaining` (which calls this).  I thought it'd be fine to leave it. It makes `get_low_stock_remaining`) a bit easier to follow and if this is needed in the future (or if the logic for getting remaining stock changes) we already have this available.